src/test/java/io/gravitee/el/spel/context/SpelTemplateEngineTest.java

```diff
@@ -692,4 +693,36 @@ public void shouldGetValueAsBoolean() {
         );
         assertFalse(engine.getValue("{#request.headers['X-Gravitee-No-Present'] != null}", Boolean.class));
     }
+
+    @Test
+    public void shouldEvaluateSimpleRegex() {
+        String pathInfo = "/user";
+        String regex = "^\\/user";
+
+        Pattern pattern = java.util.regex.Pattern.compile(regex);
+        Matcher matcher = pattern.matcher(pathInfo);
+        assertEquals(true, matcher.matches());
+
+        when(request.pathInfo()).thenReturn(pathInfo);
+        final TemplateEngine engine = TemplateEngine.templateEngine();
+        engine.getTemplateContext().setVariable("request", new EvaluableRequest(request));
+
+        engine.getValue("{#request.pathInfo.matches('" + regex + "')}", Boolean.class);
```

Comment:
Should we `assertTrue` for this call just to be sure it's working fine in an EL expression?